common/net_api.h

```diff
@@ -27,7 +27,6 @@
 // Set this flag for things like broadcast requests, etc. where the engine should not
 // kill the request hook after receiving the first response
 #define FNETAPI_MULTIPLE_RESPONSE		( 1<<0 )
-#define FNETAPI_LEGACY_PROTOCOL			( 1<<1 ) // xash3d-fwgs extension
```

Comment:
Won't it break something? Maybe keep this bit as reserved?

Reply:
Left a comment for this.